### PR TITLE
Attempt to fix QdmRequestSome implementation of QDMRequestInterface

### PR DESCRIPTION
**Pull request by kchapple**
@sjpadgett I haven't tested this update, but this should get you close. I don't think this was used previously in the system anywhere, so never got a full workout.

### CODE REVIEW

File: src/Services/Qdm/QdmRequestSome.php [hunk 1]
```diff
@@ -28,39 +24,18 @@ class QdmRequestSome implements QdmRequestInterface
      */
     public function __construct(array $pids)
     {
-        $this->pids = $pids;
-
-        if (is_array($pids)) {
-            $this->pidString = implode(",", $pids);
-        }
-
+        $filterClause = str_replace("?,", count($pids));
```
Comment (sjpadgett) [inline src/Services/Qdm/QdmRequestSome.php:27]: @kchapple Doesn't make sense. Missing subject in str_replace.

Comment (kchapple) [reply src/Services/Qdm/QdmRequestSome.php]: sorry @sjpadgett is supposed to be str_repeat(), auto-complete beat me this time.

Comment (sjpadgett) [reply src/Services/Qdm/QdmRequestSome.php]: That worked so thank you sir. I'll bring in once you update PR.

**Review by sjpadgett — APPROVED** [review]
Sorry just noticed you already update. Thanks again and merging once test complete.

**Pull request merged**